ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -404,44 +404,94 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-ovn-day2
-- as: aws-ipi-localzone-byo-subnet-ovn-f7
-  cron: 50 2 7,14,21,28 * *
+- as: aws-ipi-localzone-byo-subnet-ovn-mini-perm-f14
+  cron: 27 11 12,26 * *
   steps:
     cluster_profile: aws-1-qe
     env:
+      AWS_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
       BASE_DOMAIN: qe.devcluster.openshift.com
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-ovn
-- as: aws-ipi-localzone-byo-subnet-ovn-f28-destructive
-  cron: 53 0 27 * *
+- as: aws-ipi-localzone-byo-subnet-ovn-mini-perm-f28-destructive
+  cron: 7 3 15 * *
   steps:
     cluster_profile: aws-1-qe
     env:
+      AWS_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
       BASE_DOMAIN: qe.devcluster.openshift.com
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-ovn
-- as: aws-ipi-localzone-fips-f7
-  cron: 24 12 3,12,19,26 * *
+- as: aws-ipi-localzone-byo-subnet-sts-mini-perm-f14
```

Comment:
Will we have a following PR to add upgrade ci jobs ?

Reply:
If possible, it will be addressed in the next commit.